### DISCUSSION TITLE
HP ProBook 640 G1 (Sonoma)

### DISCUSSION THREAD

**Discussion by johnwalked** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#200] [Q&A]
I've successfully installed macOS Sonoma on my laptop if anyone here can help me, 2 things are bothering me 
1. touch pad and 
2. wifi is not detected as well as usb tethering

ps. I've a working a perfectly efi on monetary

**Comment by chris1111** [comment]
What is the wifi card?
Send EFI zip here I will see your perfect install

**Reply by johnwalked** [reply]
![Screen Shot 2023-11-11 at 11 02 44 PM](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC/assets/69463380/9b59f194-54dc-44b4-b920-7ca4957cb70d)

My wifi card is BCM943228HMB

**Reply by johnwalked** [reply]
and here is my EFI I'm using on Monterey (this has wifi and bluetooth bug only)
(https://mega.nz/file/Y1Qn0IID#oJoOdS-SNjcGaBvcQa3S8aNy16mepS73xko-fZvq0Vg)

**Comment by chris1111** [comment]
Post the EFI on ZIP here, your link is not work

**Reply by johnwalked** [reply]
sorry check now It's working

**Comment by chris1111** [comment]
This is not an EFI build from the program HP Probook Elitebook macOS
Discussion closed don't come here and waste serious people's time
